types/core.d.ts

```diff
@@ -53,10 +57,12 @@ export interface Config {
     expire?: number;
     ping?: number;
     timeout?: number;
+    session?: number;
     shutdown?: number;
     cssRules?: boolean;
     lean?: boolean;
-    tokens?: string[];
+    track?: boolean;
```

Comment:
What does this mean?

Reply:
This was a dead reference in the code for some time. This is left over from our previous discussions on ambition client side compression by sending tokens from the server. We are not going with it right now... and all related code is already out, so I also removed these references for now. In short, think of it as dead code. Will not impact backend.